CriFs.V2.Hook/Config.cs

```diff
@@ -21,6 +21,11 @@ public class Config : Configurable<Config>
     [DefaultValue(false)]
     public bool PrintFileRedirects { get; set; } = false;
     
+    [DisplayName("Print Binder Find")]
+    [Description("Prints all instances of Binder Find using the Info log level.")]
```

Comment:
This needs some additional detail.

Name
```
Print Binder Access
```

Desc
```
Prints all instances of the Binder's Find function using the Info log level.\n
This can be used to detect file access initiated from internal CRI logic.
```

The original field for file accessed should also have its description updated.

Name
```
Print File Registration
```

Desc
```
Prints when a file is registered onto the internal CRI File Loader\n
This can be useful to find out when a file is being accessed for the first time.
```

-------------------

Something of this sort.
Please doublecheck if register is called more than once per file in practice, I unfortunately had some data loss back in January, so I don't have my disassemblies anymore.